Std/Tactic/CoeExt.lean

```diff
@@ -5,10 +5,30 @@ Authors: Gabriel Ebner, Mario Carneiro
 -/
 import Lean.PrettyPrinter.Delaborator.Builtins
 import Std.Lean.Delaborator
-open Lean Meta Std
+open Lean Elab.Term Meta Std
 
 namespace Std.Tactic.Coe
 
+
+/-- `⇑ t` coerces `t` to a function. -/
+-- We increase the right precedence so this goes above most binary operators.
+-- Otherwise `⇑f = g` will parse as `⇑(f = g)`.
+elab "⇑" m:term:80 : term => do
```

Comment:
This precedence doesn't sound high enough. I would expect that `⇑f x` means `(⇑f) x` which means that the precedence needs to be higher than application.

Reply:
This precedence is copied from mathlib, evidently it was never used!

Reply:
Now merged in mathlib, so this change won't cause any issues with an Std bump.